CHANGELOG.md

```diff
@@ -27,6 +27,16 @@
   builds, `--authfile` and other authentication options, and more. See the [user
   guide](https://docs.sylabs.io/guides/latest/user-guide/build_a_container.html#dockerfile)
   for more information.
+- Docker-style SCIF containers
+  ([https://sci-f.github.io/tutorial-preview-install](https://sci-f.github.io/tutorial-preview-install))
+  are now supported. If the entrypoint of an OCI container is the `scif`
+  executable, then the `run` / `exec` / `shell` commands in `--oci` mode can be
+  given the `--app <appname>` flag, and will automatically invoke the relevant
+  SCIF command. For example:
```

Comment:
Please drop the example here. The change log should be kept relatively brief - we'll have docs and it'll be called out in the 'what's new in 4.1' section.